docs/daily-usage.md

```diff
@@ -376,6 +384,46 @@ services:
 
 </details>
 
+## Visual Studio Code Integration
+
+This project will integrate with it [Visual Studio Code](https://code.visualstudio.com/)
+to provide you with a better and easier-to-use experience through:
+
+- [Suggested extensions](/.vscode/extensions.json)
+- Launch configurations (See section on [tasks file](/tasks.py.jinja))
+- Workspace definition (See section on [tasks file](/tasks.py.jinja))
+- Integration with invoke (See section on [tasks file](/tasks.py.jinja))
+- Debugger integration (See `tasks` section on [tasks file](/tasks.py.jinja) and
+  sections below)
+
+### Debugging Python code
```

Comment:
I miss the section about debugging python + JS code, which I think is the most awesome new feature.

Reply:
Pls check if that's what you had in mind